cmd/sand-agent-cli/curl.go

```diff
@@ -34,7 +35,8 @@ func (a *App) Curl(c *cli.Context) error {
 
 	httpClient := &http.Client{
 		Transport: client.NewHTTPRoundTripper(context.Background(), c.String("network"), sand.HTTPRoundTripperOpts{
-			TLSConfig: &tlsConfig,
+			TLSConfig:         &tlsConfig,
+			DisableKeepAlives: true,
```

Comment:
You can remove this, and disable keepalived by default in the `NewHTTPRoundTripper` method

Reply:
ack

Reply:
@stopostit Just one thing, it might be time to add a timeout here though. I will add timeout on database-core too later